### ISSUE TITLE
remove Blaze wrapper around Accounts packages

### ISSUE THREAD

**Issue by kieckhafer** [reactioncommerce/reaction#2746]
Follow up to #2745 

Remove Blaze wrapper around accounts packages now in `/import/plugins/core/accounts

**Comment by spencern** [comment]
@kieckhafer is this a blocker for marketplace release?

**Comment by kieckhafer** [comment]
No, everything works as is, this is just part of our overall React conversion, which has a few other things that still need to be done as well.
